### ISSUE TITLE
Comments order in notifications

### ISSUE THREAD

**Issue by IvanRF** [websharks/comment-mail#269]
I moderate comments. I just used the option "Approve and Reply" and the comments in the notification mail are in the opposite order. In other words, the reply appears first and then the original comment:

For example, for a comment->reply->reply the mail content is:
```
In response to this comment posted by Colin: 
....
This reply was posted 19 hours ago by Ivan: 
...
------------
This comment was posted 1 day ago by Colin. 
....
------------
In response to this comment posted by Ivan: 
....
This reply was posted 13 hours ago by Colin: 
....
```

So, the comment which originated the thread is in the middle and it should be the first one.

**Comment by raamdev** [comment]
@IvanRF It's not clear from your example which comment is a top-level comment and which comment is a sub-comment. Could you label them with "Level 1", "Level 1 Reply 1", etc?

**Comment by IvanRF** [comment]
@raamdev I edited the example

**Comment by IvanRF** [comment]
@raamdev it is the first time I use the WordPress option "Approve and Reply", maybe that caused those 2 comments to be in reverse order.

**Comment by raamdev** [comment]
@IvanRF I agree, something seems off there. I'm going to mark this as something that needs testing so that we can take a closer look. Is there another way of approving and replying to comments that doesn't seem to cause the out-of-order behavior?

**Comment by IvanRF** [comment]
@raamdev I normally use the quick links for each comment. First, click "Approve" and then I use the "Reply" link. This time, I used the Reply link without previously approving, and that's when the option "Approve and Reply" appears as the button.

**Comment by renzms** [comment]
@raamdev 

Tested this using `Approve and Reply`, **WordPress 4.6.1** running **Twenty Sixteen theme** with **Comment Mail Pro Version 160818** installed. Using only default email templates.

Was unable to replicate thread seen above.

```
In response to your comment:

4th

   This reply was posted 1 minute ago by tester:

     5th

         Continue reading | Reply to tester
```

The top most comment was the oldest comment, and the lowest comment was the latest `approved and replied` to comment. A reply to a reply/approve and reply only showed the previous parent comment and the current child comment.

**Comment by raamdev** [comment]
@renzms Thanks for testing. @IvanRF Do you see anything off about how Renz is trying to reproduce the issue you were seeing? Are you still seeing this issue on your end?

**Comment by IvanRF** [comment]
@raamdev I don't normally use `Approve and Reply`, but I will try it next time.

Ouch, now that I checked my site I have 1292 elements on the Mail Queue (since 2 weeks ago), is there any way to force them to be send?
